packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -1,5 +1,236 @@
-@import "tailwindcss/base";
+@import 'tailwindcss/base';
+@import '../../global/global.pcss';
 
 :host {
   @apply font-sans font-normal;
 }
+
+.result-root {
+  display: grid;
+  justify-items: stretch;
+}
+
+.result-root > section:empty {
+  display: none;
+}
+
+/* == Image styles == */
+.result-root.image-large,
```

Comment:
Adding `display`, `image` or `density` to the result list appends classes such as `image-large` or `density-comfortable`.